.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,24 @@
+name: Run benchmarks
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, reopened]
+
+# Only trigger the benchmark job when you add `run benchmark` label to the PR
+jobs:
+  Benchmark:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'run benchmark')
```

Comment:
Can you show/describe how this works? It seems to me from these lines that you need to add `run benchmark` somewhere to see this working.

Reply:
> Can you show/describe how this works?

BenchmarkCI crates `Benchmark-result` branch and saves results there.
For example look at [this PR](https://github.com/tkf/BenchmarkCI.jl/pull/85), github-actions bot comments the results there, and saves them in `Benchmark-result` branch [here](https://github.com/tkf/BenchmarkCI.jl/tree/benchmark-results) (In this repositories case we need not mention `run benchmark` explicitly)

There are many ways one can trigger Benchmarking. In our case when some one adds `run benchmark` label to the comment of PR. We can also have configuration that trigger benchmarking for each PR without mentioning it explicitly.